src/scr_util.c

```diff
@@ -388,3 +388,56 @@ spath* scr_get_prefix(const char* str)
 
   return prefix_path;
 }
+
+/* Clone of apsrintf().  See the standard asprintf() man page for details */
+int asprintf(char** strp, const char* fmt, ...)
```

Comment:
It looks like all of the ``asprintf`` functions are for construction of path strings.  Let's drop ``asprintf`` and use ``spath`` for those instead.